sites/public/src/components/listings/search/ListingsSearchCombined.tsx

```diff
@@ -11,6 +11,7 @@ import {
   t,
 } from "@bloom-housing/doorway-ui-components"
 import { FormOption, ListingsSearchModal } from "./ListingsSearchModal"
+import { jurisdiction } from "@bloom-housing/shared-helpers/__tests__/testHelpers"
```

Comment:
I realize that we're down to the wire with getting these changes finalized, but it makes me a bit uneasy to use a dummy jurisdiction in testHelpers in place of the actual value from the database.  Can we just call `fetchJurisdictionByName` in `getServerSideProps` and pass it in that way?

Reply:
I'm getting `  message: 'Not Found',` from the backend when calling

```
  const jurisdiction = fetchJurisdictionByName(        
    runtimeConfig.getBackendApiBase(),                                                                                                                                            
    runtimeConfig.getJurisdictionName()                                                                                                                                                             
  ) 
```
in dev. When we reseed in dev, we're not adding the "Bay Area" jurisdiction, that I can see.

Reply:
nm. I see it exists but my public site isn't sending it at the moment.